examples/tv-casting-app/tv-casting-common/support/CastingStore.cpp

```diff
@@ -470,6 +475,7 @@ CHIP_ERROR CastingStore::WriteAll(std::vector<core::CastingPlayer> castingPlayer
 
     for (auto & castingPlayer : castingPlayers)
     {
+        ChipLogProgress(AppServer, "CastingStore::WriteAll() writing castingPlayer:");
```

Comment:
At line 461, you could just add logging of the size() of castingPlayers, and avoid logging this here?

Reply:
For line 478: ChipLogProgress(AppServer, “CastingStore::WriteAll() writing castingPlayer:“);
Since we’re logging multiple casting players in a loop, it helps to divide the player info (endpoints and server list) into sections